src/officehours_ui/static/css/main.css

```diff
@@ -321,14 +321,20 @@ a.nav-link.disabled:hover, a.nav-link.disabled:active, a.nav-link.disabled:focus
 	color: #ffffff;
 }
 
+.btn-success {
+	color: #ffffff;
+	background-color: #00820d;
+	border-color: #00820d;
```

Comment:
I modified the default Bootstrap button success color to be a darker shade of green (`#00820d`), resulting in a constrast ratio of 5 to 1 with white. I didn't want to too much darker, since I felt those shades look different enough to maybe be confusing or jarring to users' familiar with (or subconsciously familiar with) the Bootstrap palette. Thoughts, @teshsopher? I am open to feedback on this.

Reply:
That's perfect!

Reply:
Cool, thanks